app/models/order.rb

```diff
@@ -263,6 +265,35 @@ def close_direct!(user)
     update_attributes! state: 'closed', updated_by: user
   end
 
+  def send_to_supplier(user)
```

Comment:
Does this warrant an exclamation mark? (Since it sends a mail and calls `update!`, which changes the order).